Documentation/ProjectFile/prj/processes/process/TWOPHASE_FLOW_PP/t_temperature.md

```diff
@@ -0,0 +1 @@
+Constant temperature (K) for the simulation.
```

Comment:
Please clarify if this is an initial, or reference, or other kind of temperature. Where is it used, what is influenced by this setting?